image/tarexport/tarexport.go

```diff
@@ -25,7 +25,7 @@ type manifestItem struct {
 
 type tarexporter struct {
 	is             image.Store
-	ls             layer.Store
+	lss            map[string]layer.Store
```

Comment:
Maybe just expose accessing specific layerstore through the imagestore so there is no duplicate map.